third_party/intellij-plugins-dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcessZ.java

```diff
@@ -198,9 +198,6 @@ public void logInformation(final String message, final Throwable exception) {
     });
   }
 
-  public void scheduleConnect() {
```

Comment:
I wondered the same thing but there's no parent implementation --- the closest is `scheduleConnect(String)` which has a different signature obviously and is private.